e2e_tests/tests/cluster/test_exp_continue.py

```diff
@@ -18,13 +18,13 @@ def test_continue_config_file_cli() -> None:
         sess,
```

Comment:
this test doesn't seem like it adds much more value on top of the one below it. can we just delete it?

Reply:
actually, i just looked at the whole file. most of the tests here seem like overkill... also, we're gonna have to delete/rewrite "continue" tests anyway since post-searcher-context continue will be resume.

that'll be in a separate PR tho so nvm.